src/BasisSystems/ACE/ace.jl

```diff
@@ -0,0 +1,105 @@
+using JuLIP: Atoms, JVec, JMat, site_energy, Ref
+using JuLIP.Chemistry: AtomicNumber
+import ACE1
+using Unitful, UnitfulAtomic
+
+struct ACE <: BasisSystem
+    species           :: Vector{Symbol}
+    body_order        :: Int   
+    polynomial_degree :: Int
+    wL                :: Real
+    csp               :: Real
+    r0                :: Real 
+    rcutoff           :: Real
+    rpib              :: ACE1.RPIBasis
+end 
+
+function ACE(species, body_order, polynomial_degree, wL, csp, r0, rcutoff)
```

Comment:
This should be changed to ACE(species, body_order, polynomial_degree, rcutoff; wL=1.5, csp=1.0, r0=2.5)
Actually, I'm not a 100% sure what the default r0 should be, but regardless, I think it should typically not be set.